mitodl/mitxpro
# Make course/program enrollment foreign key to company blankable in Django Admin

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #553

#### What's this PR do?
Adds `blank=True` to `CourseEnrollment` and `ProgramEnrollment` models

#### How should this be manually tested?
In Django admin, create or edit a `CourseEnrollment` and `ProgramEnrollment`.  Leave `company` blank and save.  It should work.
